src/lib/core/overlay/overlay-state.ts

```diff
@@ -12,10 +12,16 @@ export class OverlayState {
   /** Whether the overlay has a backdrop. */
   hasBackdrop: boolean = false;
 
+  /** Custom class to add to the backdrop **/
   backdropClass: string = 'md-overlay-dark-backdrop';
 
+  /** The width of the overlay panel **/
+  width: number;
+
+  /** The height of the overlay panel **/
+  height: number;
```

Comment:
We should support percent with this as well. I'd make the type `number|string`, assume numbers are `px` and strings are whole units.